app/graph/mutations/explainer_item_mutations.rb

```diff
@@ -7,5 +7,18 @@ class Create < Mutations::CreateMutation
     argument :project_media_id, GraphQL::Types::Int, required: true
   end
 
+  class SendExplainersToPreviousRequests < Mutations::BaseMutation
+    argument :id, GraphQL::Types::Int, required: true
```

Comment:
For consistency, we've been using `dbid` for the numeric incremental database ID and `id` for the Base 64 GraphQL ID.

Reply:
used `dbid`